buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/DistributedPerformanceTest.groovy

```diff
@@ -45,12 +45,13 @@ import java.util.zip.ZipInputStream
  * blocks until all the jobs have finished and aggregates their status.
  */
 @CompileStatic
+@CacheableTask
 class DistributedPerformanceTest extends PerformanceTest {
```

Comment:
Are the templates we use for the test project already an input? What about the baseline(s)?
For the baselines, I think `nightly` would need some special treatment if it doesn't have already.

Reply:
This is the performance test coordinator, which schedules the real work to performance test workers, so it has nothing to do with the template and the baseline. Basically, the idea is, "if no production code changes, I don't need to rerun any performance test".

Reply:
You're right, templates and baseline should also be an input.

Reply:
You can pass baselines as an argument, which overwrites the baseline specified in the test. If the baseline is `nightly`, caching needs to be turned off.